charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -1,6 +1,4 @@
 {{- if .Values.controller.ingressClassResource.enabled -}}
-# We don't support namespaced ingressClass yet
```

Comment:
Removing this line vaguely removes the confirmation that we don't support namespace scoped ingressClass. Wonder if this is desired. It helped in some issues to state that we don't support namespace scoped installs and hence don't support namespace scoped ingressClasses.

Reply:
You're right, gonna remove that. I wasn't sure if someone ever read it that in the code. Should we maybe move it to the values and this way also to the `README.md`?

Reply:
Yes, values and README are a good place to have that info explicit. (unrelated note - similar treatment to the feature of creating internal lb in the same install instance as a external lb)

Reply:
Just pushed another commit tackling that.